portal/src/main/webapp/WEB-INF/jsp/visualize.jsp

```diff
@@ -45,6 +45,14 @@ window.maxTreeDepth = '<%=GlobalProperties.getMaxTreeDepth()%>';
 window.skinExampleStudyQueries = '<%=GlobalProperties.getExampleStudyQueries().replace("\n","\\n")%>'.split("\n");
 
 window.priorityStudies = {};
+
+// global variables required for MutationMapper annotation column
+window.showCivic = <%=GlobalProperties.showCivic()%>;
```

Comment:
I think, for now, we should move the global variables related to mutation mapper into a separate jsp file. Then we can include the same jsp file in the cross cancer page as well.